ansible/roles/machine/setup/tasks/install_packages.yml

```diff
@@ -36,13 +36,19 @@
     - PyYAML  # ipa-vagrant-ci dependency
     - vim
     - NetworkManager
-    - python3-paramiko
 
 - name: install pip dependencies
   pip:
+    executable: pip3
     name: pytest-html
     state: latest
 
+- name: install paramiko
+  pip:
+    executable: pip3
+    name: paramiko
+    version: 2.2.1
```

Comment:
Why do we require this specific version?

Reply:
I see you mentioned the issue in the commit message. Could you please link the issue with a comment in the code as well?

Reply:
Sure.